web/src/hooks/model-deployments/useDeploymentsData.jsx

```
@@ -160,114 +156,127 @@ export const useDeploymentsData = () => {
+      applyDeploymentsData({ data, page });
     } catch (error) {
-      console.error('Search error:', error);
-      showError(t('搜索失败'));
+      if (seq !== requestSeq.current) return;
+      console.error(error);
+      showError(isSearchMode ? t('搜索失败') : t('获取部署列表失败'));
       setDeployments([]);
+      setDeploymentCount(0);
+    } finally {
+      if (seq !== requestSeq.current) return;
+      setLoading(false);
```

Biome (2.1.2)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)